requirements-test.txt

```diff
@@ -1,2 +1,3 @@
 pytest==2.9.2
 py==1.4.31
+pygments==2.1.3
```

Comment:
I couldn't get the tests to run locally without it. It can be revert if
I've missed something?

On 1 August 2016 at 19:26, James Saryerwinnie <notifications@github.com>
wrote:

> In requirements-test.txt
> <https://github.com/awslabs/chalice/pull/75#discussion_r73027646>:
>
> > @@ -1,2 +1,3 @@
> >  pytest==2.9.2
> >  py==1.4.31
> > +pygments==2.1.3
>
> Was this intentional?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/awslabs/chalice/pull/75/files/fd42f38a4ab7869e04bbb28d3c99010343563fcb#r73027646>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGKdwsABbPTmARMaB4cyDrJs7pGojm98ks5qbjppgaJpZM4JYx-y>
> .
>